Automattic/wp-calypso
# Update/onboarding import tracking

#### Changes proposed in this Pull Request
Changes provide additional logic to call `recordTracksEvent` methods on the places where are missing, defined by [the Figma doc](https://www.figma.com/file/yKn2XDQWEnAlYksYvpbXOZ/Importer-Flow-Metrics?node-id=0%3A1).

#### Testing instructions
* Go to `/start/importer/capture?siteSlug={YOUR_SITE}.wordpress.com`
* Pass through the onboarding import flow
* Check if tracking is active
* You can use [Tracks Vigilante chrome plugin](https://github.com/Automattic/tracks-chrome-extension) for easier follow

Related to #57131
